kong/templates/kong_defaults.lua

```diff
@@ -42,6 +42,7 @@ cluster_max_payload = 16777216
 cluster_use_proxy = off
 cluster_dp_labels = NONE
 cluster_rpc = off
+cluster_cjson = off
```

Comment:
Should be `true`?

Reply:
The default value `off` of `cluster_cjson` means than we will enable simdjson in clustering by default.